clusterloader2/pkg/measurement/common/network/manifests/worker-deployment.yaml

```diff
@@ -20,6 +20,9 @@ spec:
       - image: gcr.io/k8s-testimages/netperfbenchmark:0.1
```

Comment:
You will probably need to create new image and push it ;)

Reply:
@wojtek-t could you please help in creating new image after this is committed?

Reply:
I can help you with it ;)

Reply:
Ok thanks!:)